connectors/aws/aws-sns/src/main/java/io/camunda/connector/sns/inbound/SnsWebhookExecutable.java

```diff
@@ -12,6 +12,7 @@
 import com.amazonaws.services.sns.message.SnsSubscriptionConfirmation;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import io.camunda.connector.api.annotation.InboundConnector;
+import io.camunda.connector.api.inbound.Health;
```

Comment:
@chillleader Thanks for these changes!

I suggest we keep this PR open until I confirm with Sara whether QA is required. If QA is required, then I think we should only merge once QA is complete.

Reply:
As discussed in Slack, @slolatte is ok with us merging in to conduct QA.

We can't use Connectors preview environments here because they only work with Self-Managed, but inbound connectors are a SaaS-only feature.